README.md

```diff
@@ -1,25 +1,91 @@
-# psibase
+<div align="center">
+  <img src="https://about.psibase.io/assets/logo-psibase-green.0714b8ee.svg" />
+</div>
+<h1 align="center">
+  psibase
+</h1>
 
-This repo contains the sources to
+## What is psibase
 
-- `psinode`, the psibase node
-- `psibase`, the command-line utility for pushing transactions and booting a chain
-- psibase services
-- `psidk`, the service development kit
+Psibase is a powerful protocol that enables communities to come together to easily self-host web applications. This repository is an open source implementation of that protocol.
+Any psinode deployment comes with some default applications. Some of these are mandatory and critical for the infrastructure to function properly, and others are optional example user applications.
 
-## About Page, Documentation, and Demo
+Primary packages included in this repository:
 
-The [Demonstration Chain](https://psibase.io/) hosts both the [About Page](https://about.psibase.io/) and the [Documentation](https://doc-sys.psibase.io/).
+| Package   | Description                                                                   |
+|-----------|-------------------------------------------------------------------------------|
+| `psinode` | The psibase infrastructure node                                               |
+| `psibase` | Command-line utility for interacting with psinode. Used primarily by scripts. |
+| `psidk`   | The SDK used for developing web services                                      |
 
-## Binary Release
+-------
 
-If you want to operate a node, build services, or explore psibase's command-line utility, we recommend installing a [Binary Release](https://github.com/gofractally/psibase/releases). We provide a single binary package which should run on most recent Linux distributions. See the [installation instructions](https://doc-sys.psibase.io/linux.html).
+<p align="center">
+    &bull;
+    <a href="#demo-deployment-%EF%B8%8F">Demo 🖥️</a> &bull;
+    <a href="#technical-docs-">Technical docs 📓</a> &bull;
+    <a href="#running-psibase-">Running psibase 🚀</a> &bull;
+    <a href="#build-from-source-%EF%B8%8F">Build from source 🛠️</a> &bull;
+    <a href="#contribute-">Contribute 🧬</a> &bull;
+    <a href="#support-">Support 📱</a> &bull;
+</p>
```

Comment:
I would expect a horizontal navbar to be above everything but the logo and title.

Reply:
Done